service-api/src/main/java/greencity/dto/RangeDto.java

```diff
@@ -0,0 +1,15 @@
+package greencity.dto;
+
+import lombok.*;
+
+@Getter
+@Setter
```

Comment:
not a good practice to havre both builder and setter. You can achieve immutability if you avoid usage of setters here. So your object will be built using Builder only. Also, please review if there are other redundant annotations.